doc/source/data/working-with-images.rst

```diff
@@ -0,0 +1,267 @@
+.. _working_with_images:
+
+Working with Images
+===================
+
+With Ray Data, you can easily read and transform large image datasets.
+
+This guide shows you how to:
+
+* :ref:`Read images <reading_images>`
+* :ref:`Transform images <transforming_images>`
+* :ref:`Perform inference on images <performing_inference_on_images>`
+* :ref:`Save images <saving_images>`
+
+.. _reading_images:
+
+Reading images
+--------------
+
+Ray Data can read images from a variety of formats.
+
+To view the full list of supported file formats, see the
+:ref:`Input/Output reference <input-output>`.
+
+.. tab-set::
+
+    .. tab-item:: Raw images
```

Comment:
I would clarify this as "Ray images (JPG, PNG, etc.)" for full clarification

Reply:
Tried this but felt that the tab text was too verbose. Clarified in the content instead.

Lemme know what you think.

Reply:
We could also rename "Raw images" as "Image files"